### DISCUSSION TITLE
[Detector Support]: frigate live view detected preview 1 hour ago  only？

### DISCUSSION THREAD

**Discussion by asinwang** [blakeblackshear/frigate#13635] [Ask A Question]
### Describe the problem you are having

frigate live view detected preview 1 hour ago  only？can show all detected  current day

### Version

v0.14.1

### Frigate config file

```yaml
none
```

### docker-compose file or Docker CLI command

```yaml
none
```

### Relevant Frigate log output

```shell
none
```

### Relevant go2rtc log output

```shell
none
```

### Operating system

Other Linux

### Install method

Docker Compose

### Object Detector

OpenVino

### Screenshots of the Frigate UI's System metrics pages

none

**Comment by NickM-27** [comment]
the main page is for showing "what is happening or happened recently on my cameras", the review page should be used to review older review items

**Reply by asinwang** [reply]
Yes,  now displayed 1 hour ago. Is it possible to set this time or the number of displays?
